.github/workflows/ci.yml

```diff
@@ -38,19 +34,6 @@ jobs:
         python -m pip install --upgrade pip
         pip install protobuf==3.7.1
 
-    # Cache yarn deps. thx https://github.com/actions/cache/blob/master/examples.md#node---yarn
```

Comment:
looking at the runs for https://github.com/GoogleChrome/lighthouse/pull/10988/checks?check_run_id=783384854 and running a bunch of them myself, it seems like caching only saves like 5 seconds out of ~30. Doesn't seem worth it.